default_config/dev/config.yaml

```diff
@@ -23,11 +23,14 @@ DISCOVERY:
     PROGRAM_DETAIL_TEMPLATE: ''
 
 PROGRAM:
-  TYPE: 'native'
+  TYPE: 'webview'
   WEBVIEW:
     PROGRAM_URL: ''
     PROGRAM_DETAIL_URL_TEMPLATE: ''
 
+DASHBOARD:
+  TYPE: 'primary_course'
```

Comment:
The first screen that the user sees is the PrimaryCourseFragment, and we don't have any grids on that screen. In my opinion, the "primary_course" is better because it is the main part of the screen, and the AllEnrolledCoursesFragment is not the main screen in that flow.

Reply:
We do have other courses cards on PrimaryCourseFragment as well which makes it a grid view. or more specifically we can call it staggered grid view.